src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -184,21 +184,21 @@ default String checkClazz(String expectedClass, double waitFor, double timeTook)
      * @param timeTook      - the amount of time it took for wait for something (assuming we had to wait)
      * @return String: the attribute value of the attribute specified from the element. null will be returned if the element isn't present
      */
-    default String checkAttribute(String attribute, String expectedValue, double waitFor, double timeTook) {
+    String checkAttribute(String attribute, String expectedValue, double waitFor, double timeTook) {
         // get the actual attribute value
-        String elementValue = getElement().get().attribute(attribute);
+        String elementValue = this.element.get().attribute(attribute);
         // record the result
         if (elementValue == null) {
-            getReporter().fail(getElement().prettyOutput() + " having an attribute of <i>" + attribute + " with a value of <b>" +
-                    expectedValue + "</b>", waitFor, getElement().prettyOutputStart() + " does not have an attribute of <i>" + attribute + "</i>", timeTook);
+            this.reporter.fail(this.element.prettyOutput() + " having an attribute of <i>" + attribute + " with a value of <b>" +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " having an attribute of <i>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal " with a value of <b>" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)